test/Billing.Test/Controllers/FreshdeskControllerTests.cs

```diff
@@ -1,119 +1,67 @@
-﻿using System.Text.Json;
-using Bit.Billing.Controllers;
+﻿using Bit.Billing.Controllers;
+using Bit.Billing.Models;
 using Bit.Core.Entities;
 using Bit.Core.Repositories;
-using Bit.Core.Settings;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using NSubstitute;
 using Xunit;
 
 namespace Bit.Billing.Test.Controllers
 {
+    [ControllerCustomize(typeof(FreshdeskController))]
+    [SutProviderCustomize]
     public class FreshdeskControllerTests
     {
         private const string ApiKey = "TESTFRESHDESKAPIKEY";
-        private const string WebhookKey = "TEST_FRESHDESK_WEBHOOKKEY";
-        private const string TicketId = "TEST_FRESHDESK_TICKETID";
-        private const string Email = "TEST@EMAIL.COM";
+        private const string WebhookKey = "TESTKEY";
+
+        [Theory]
+        [BitAutoData((string)null, null)]
+        [BitAutoData((string)null)]
+        [BitAutoData((FreshdeskWebhookModel)null)]
```

Comment:
Are you sure this works? I believe this is just passing null in for the first parameter both times. I don't remember writing in anything about type matching for specified parameters -- good idea, though!

Reply:
I thought it did but it seems not. I've changed the code to pass a string value as first parameter and null value as second.